.gitignore

```diff
@@ -25,3 +25,10 @@ requirements.txt
 # compiled stuff
 out/
 **/*.log
+**/curl
+**/json-c
+**/qmstr
+**/*.json
```

Comment:
This is not supposed to be here. I guess this should be in the demo repo.

Reply:
This is due to the documentation. If we try to run the documentation locally these directories are created.